config/docs/options_README.md

```diff
@@ -0,0 +1,264 @@
+e.g.* Metadata_Cells_ObjectNumber  
+
+`merge_columns:`
+*CellProfiler outputs measurements for each compartment in a separate .csv.
+In this step we merge them together using the merge_columns.*  
+`image_column:` *The name of the column that describes the image number.  
+e.g.* ImageNumber  
+`linking_compartment:` *The compartment used to link other compartments.  
+e.g.* Cytoplasm  
+`linking_columns:`*Column names that link your parent compartment to other compartments.  
+You will need to set a column name for each compartment set in* `core: compartments`*.  
+The column names listed must also exist in the data for the compartment listed in* `linking_compartment`*.*  
+`cells:` *e.g.* Metadata_Cytoplasm_Parent_Cells  
+`nuclei:` *e.g.* Metadata_Cytoplasm_Parent_Nuclei  
+`metadata_merge_columns:`  
+*These are the columns that are used to link foci data to compartment data.*  
+`foci_cols:` *The columns that describe the image number and the cell number in the foci data.  
+e.g.*  
+    - Metadata_Foci_ImageNumber  
+    - Metadata_Foci_Parent_Cells  
+`cell_cols:`*The columns that describe the image number and the cell number in the cell data.  
+e.g.*  
+    - Metadata_Cells_ImageNumber  
+    - Metadata_Cells_ObjectNumber  
+`foci_site_col:`*The column in the foci data that contains the site number.  
+e.g.* Metadata_Foci_site
+
+`summarize-cells:`  
+`perform:` *Do you want to perform 3.visualize-cell-summary step?   
+Set to* true *or* false.  
+`force_overwrite:` *Do you want to overwrite any existing data?  
+Set to* true *or* false.
+
+
+`summarize-plate:`   
+`perform:` *Do you want to perform 4.image-and-segmentation-qc step?  
+Set to* true *or* false.  
+`force_overwrite:` *Do you want to overwrite any existing data?  
+Set to* true *or* false.  
+`correlation_threshold:` *Set the minimum correlation for two images to have to pass the correlation filter.  
+e.g.* .2  
+`painting_image_names:` *The names of all input images used for Cell Painting as they are named in CellProfiler.
+You can find image names as part of any measurement column in Image.csv (e.g. columns starting with Width_).  
+e.g.*  
+    - ConA  
+    - Hoechst  
+    - Mito  
+    - SYTO  
+    - WGA  
+`barcoding_prefix:` *The prefix at the start of all input images used for SBS as they are named in CellProfiler.
+You can find image names as part of any measurement column in Image.csv (e.g. columns starting with Width_)  
+e.g.* CorrCycle
+
+## profile:
+*Configuration specific to the steps in 1.generate-profiles.*
+
+`single_cell:`  
+`perform:` *Do you want to perform 0.merge_single_cells step?  
+Set to* true *or* false.  
+
+`force_overwrite:` *Do you want to overwrite any existing data?  
+Set to* true *or* false.  
+
+`prefilter_features:` *Do you want to use the prefilter file created during preprocessing?
+Set to* true *or* false.  
+
+`output_one_single_cell_file_only:` *Do you want to output a single file SAYMORE.
```

Comment:
@gwaygenomics

Reply:
If all single cell files should be concatenated into a single file rather than having site-specific single cell files.